src/code/client.js

```diff
@@ -194,21 +189,19 @@ class CloudFileManagerClient {
   }
 
   setProviderOptions(name, newOptions) {
-    return (() => {
-      const result = []
-      for (let provider of Array.from(this.state.availableProviders)) {
-        if (provider.name === name) {
-          if (provider.options == null) { provider.options = {} }
-          for (let key in newOptions) {
-            provider.options[key] = newOptions[key]
-          }
-          break
-        } else {
-          result.push(undefined)
+    const result = []
```

Comment:
I think you can just get rid of the result return.  That was just there because everything is a expression in CoffeeScript.